src/NzbDrone.Core/Extras/Subtitles/SubtitleService.cs

```diff
@@ -175,16 +183,18 @@ public override IEnumerable<ExtraFile> ImportFiles(LocalEpisode localEpisode, Ep
                 }
             }
 
-            var subtitleFiles = new List<Tuple<string, Language, string>>();
+            var subtitleFiles = new List<Tuple<string, Language, string, IEnumerable<string>>>();
 
             foreach (string file in matchingFiles)
             {
                 var language = LanguageParser.ParseSubtitleLanguage(file);
                 var extension = Path.GetExtension(file);
-                subtitleFiles.Add(new Tuple<string, Language, string>(file, language, extension));
+                var languageTags = LanguageParser.ParseLanguageTags(file);
+
+                subtitleFiles.Add(new Tuple<string, Language, string, IEnumerable<string>>(file, language, extension, languageTags));
             }
 
-            var groupedSubtitleFiles = subtitleFiles.GroupBy(s => s.Item2 + s.Item3).ToList();
+            var groupedSubtitleFiles = subtitleFiles.GroupBy(s => s.Item2 + s.Item3 + s.Item4.Aggregate(string.Empty, (tags, tag) => tags + tag)).ToList();
```

Comment:
This group-by is pretty rough as a Tuple, as mentioned above, convert to the return value to a class.

Reply:
Changed to class - https://github.com/Sonarr/Sonarr/commit/f2df494f42333ff5d57baf3c5e5d620cf045c3d8